src/Rezzza/AliceExtension/Doctrine/ORMPersister.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Rezzza\AliceExtension\Doctrine;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Nelmio\Alice\ORM\Doctrine;
+
+class ORMPersister extends Doctrine
```

Comment:
Good catch !

Unfortunately we have type hinted on `Nelmio\Alice\ORM\Doctrine`

```php
use Nelmio\Alice\ORM\Doctrine as ORMPersister;
```

[Adapter/ORM/ORMSubscriberFactory](https://github.com/rezzza/alice-extension/blob/49009145787559225a137c101bf898d76b950fb5/src/Rezzza/AliceExtension/Adapter/ORM/ORMSubscriberFactory.php#L7)
and here 
[Adapter/ORM/ORMEventSubscriber](https://github.com/rezzza/alice-extension/blob/master/src/Rezzza/AliceExtension/Adapter/ORM/ORMEventSubscriber.php#L11-L11)